apps/gitness/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "gitness",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "build:css": "npx tailwindcss -i ./src/styles.css -o ./public/styles.css",
+    "lint": "eslint .",
+    "preview": "vite preview",
+    "preinstall": "npx only-allow pnpm",
+    "pre-commit": "lint-staged",
+    "postservices": "prettier --write src/services/**/*.tsx",
```

Comment:
i think this can be removed as we are not using restful-react anymore

Reply:
Done